api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -517,6 +507,20 @@ export async function downloadDocsAndUpsertFHIR({
     await sleepBetweenChunks();
   }
 
+  await updateDocQuery({
+    patient: { id: patient.id, cxId: patient.cxId },
+    downloadProgress: { status: "completed" },
+    convertibleDownloadErrors: errorCountConvertible,
+  });
+  // send webhook to CXs when docs are done downloading
+  processPatientDocumentRequest(
+    organization.cxId,
+    patient.id,
+    MAPIWebhookType.documentDownload,
+    MAPIWebhookStatus.completed,
+    toDTO(docsNewLocation)
+  );
```

Comment:
not triggering `convert` WH update when nothing to convert bc it will happen on `updateDocQuery`